frontend/app/routes/$lang+/_protected+/alerts+/subscribe+/confirm.tsx

```diff
@@ -44,17 +47,45 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const meta = { title: t('gcweb:meta.title.template', { title: t('alerts:confirm.page-title') }) };
 
   instrumentationService.countHttpStatus('alerts.confirm', 302);
-  return json({ csrfToken, meta }); //TODO get the language and email address entered by the user when they entered their information on the index route...
+
+  const userInfoToken: UserinfoToken = session.get('userInfoToken');
+  const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin ?? '');
+  session.set('alertSubscription', alertSubscription);
+
+  return json({ csrfToken, meta, alertSubscription, userInfoToken }); //TODO get the language and email address entered by the user when they entered their information on the index route...
 }
 
 export async function action({ context: { session }, params, request }: ActionFunctionArgs) {
   const formData = await request.formData();
+  //const userInfoToken: UserinfoToken = session.get('userInfoToken');
+  //const alertSubscription = session.get('alertSubscription');
   const action = formData.get('action');
+  const instrumentationService = getInstrumentationService();
+  //const t = await getFixedT(request, handle.i18nNamespaces);
```

Comment:
Can we remove these comments or do they serve a purpose? If it's the latter, consider adding a `TODO` comment.

Reply:
removed